awesome_sso/store/minio.py

```diff
@@ -40,9 +40,25 @@ def __init__(
         else:
             self.logger.info("bucket '%s' exists", self.bucket)
 
+    def list_buckets(self):
+        """List information of all accessible buckets with text."""
+        return [x.name for x in self.client.list_buckets()]
+
+    def list_objects(self, prefix: str = None, recursive: bool = False):
+        """Lists object information of a bucket with text."""
+        return [
+            x.object_name
+            for x in self.client.list_objects(
+                self.bucket, prefix=prefix, recursive=recursive
+            )
+        ]
+
     def fput(self, name: str, file_path: str):
+        """Uploads data from a file/folder to an object in a bucket."""
         if path.isdir(file_path):
             for local_file in glob.glob(file_path + "/**"):
+                if local_file in ["__pycache__"]:
```

Comment:
1. if your local_file is "/a/b/c/__pycache__", will it be ignored? perhaps `local_file.name in ...` is what you are looking for?
2. It is a good idea to have some meaningless file being ignored, and perhaps it would be good to have this as a parameter of the function, since this is a library. Perhaps adding a parameter `exclude_files: List[str] = []` as a parameter.

Reply:
yeah, I think the second suggestion is great.